compat/v0/compat.go

```diff
@@ -68,6 +68,87 @@ func ShardsResponseFromState(state jsonrpc.ResponseQueryState) (ResponseQuerySha
 	return resp, nil
 }
 
+// ShardsResponseFromState takes a QueryState rpc response and converts it into a QueryShards rpc response
+// It can be a standalone function as it has no dependencies
+func QueryFeesResponseFromState(state jsonrpc.ResponseQueryState) (ResponseQueryFees, error) {
+	bitcoinCap, ok := state.State[multichain.Bitcoin].Get("gasCap").(pack.U64)
+	if !ok {
+		fmt.Printf("state: %v", state.State)
+		return ResponseQueryFees{},
+			fmt.Errorf("unexpected type for bitcoinCap: expected pack.String , got %v",
```

Comment:
```suggestion
			fmt.Errorf("unexpected type for bitcoinCap: expected pack.U64, got %v",
```

This should be `pack.U64` right?